src/modules/i18n/locales/de.json

```diff
@@ -76,7 +76,7 @@
     "connect": "Verbinden",
     "contact_us": "Kontakt",
     "contextpopup_lv03_url": "https://www.swisstopo.admin.ch/de/landestriangulation-lv03",
-    "contextpopup_lv95_url": "https://www.swisstopo.admin.ch/de/landesvermessung-lv95",
```

Comment:
The previous link works and the new one is not pointing to LV95 description, why changing those links ?

Reply:
This is a suggestion from @mariokeusen . We are redirecting to the description of the swiss coordinate system as a whole, and the page describes the lv95 system quite well.

Reply:
Then ok for me